db/migrations/000020_add_helmfile_and_fc_develop_ref_fields.up.sql

```diff
@@ -0,0 +1,17 @@
+alter table v2_chart_releases
+    add if not exists firecloud_develop_ref text;
+
+alter table v2_changesets
+    add if not exists from_firecloud_develop_ref text;
+
+alter table v2_changesets
+    add if not exists to_firecloud_develop_ref text;
+
+alter table v2_clusters
+    add if not exists helmfile_ref text NOT NULL;
```

Comment:
without a default here, this will fail on database rows that already exist before the migration happens

Reply:
i wonder if there's a way to add a column and use a default during that add operation but not leave the default configuration laying around in the sql afterwards. like we want a one time "put HEAD here in order to add the column"

Reply:
hmm, I bet there is a way to do that

Reply:
I've thought of plenty of hacky ways to do this but ultimately we should probably just set a default.

Reply:
[this should work](https://github.com/broadinstitute/sherlock/pull/75/commits/4d75eb432737150aa33dd9f054aa5d8cc0eae2f8)